src/console_op/health.go

```diff
@@ -34,6 +34,29 @@ import (
 	"net/http"
 )
 
+type HealthService interface {
```

Comment:
The 'doHealth', 'doLiveness' and 'doReadiness' calls are really the only 'health' calls. The rest are assorted http interface calls that got lumped here due to tight deadlines and the fact they are documented / user facing functions. The rest are more of a debugging api. Now might be the time to break them out into a new file.